configs/pvt/retinanet_pvt_l_fpn_1x_coco.py

```diff
@@ -0,0 +1,15 @@
+_base_ = [
+    'retinanet_pvt_t_fpn_1x_coco.py',
+]
+model = dict(
+    backbone=dict(
+        num_layers=[3, 8, 27, 3],
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint='https://github.com/whai362/PVT/'
+            'releases/download/v2/pvt_large.pth')))
+# optimizer
+optimizer = dict(
+    _delete_=True, type='AdamW', lr=0.0001 / 1.4, weight_decay=0.0001)
+# dataset settings
+data = dict(samples_per_gpu=1, workers_per_gpu=1)
```

Comment:
Are these settings customized for pvt-l?

Reply:
yes